examples/ads.amp.html

```diff
@@ -1463,6 +1463,12 @@ <h2>Trugaze</h2>
       json='{"targeting":{"target":["sample"],"pos":["amp"]}}'>
   </amp-ad>
 
+  <h2>UAS</h2>
```

Comment:
Please update the form [here](https://github.com/ampproject/amphtml/blob/master/examples/ads.amp.html#L56).  So it's easier for us to find your example. Thanks.